internals/scripts/InstallPyodide.js

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
```

Comment:
@jdpigeon i'm getting an import error when this postinstall script runs. any clue to why? it mentions the following:

```bash
(node:41437) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/Users/tbrooks/codespace/BrainWaves/internals/scripts/InstallPyodide.js:1
import chalk from 'chalk';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1054:16)
    at Module._compile (internal/modules/cjs/loader.js:1102:27)
    at loader (/Users/tbrooks/codespace/BrainWaves/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.<computed> [as .js] (/Users/tbrooks/codespace/BrainWaves/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
error Command failed with exit code 1.
```